<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDlgClass</class>
 <widget class="QDialog" name="OptionDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btFont">
     <property name="toolTip">
      <string>Change the font of the application</string>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btRebuiltFulltext">
     <property name="toolTip">
      <string>Rebuilt fulltext search indexes</string>
     </property>
     <property name="text">
      <string>Rebuilt fulltext search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btClearCache">
     <property name="toolTip">
      <string>Clear the local cache of online thesaurus</string>
     </property>
     <property name="text">
      <string>Clear thesaurus cache</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkExportToBibFixer">
       <property name="text">
        <string>Export to BibFixer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leBibFixerPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btBibFixerPath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkKeepAttachments">
     <property name="toolTip">
      <string>Attachments will not be delete if checked</string>
     </property>
     <property name="text">
      <string>Keep the attachments when a paper is deleted</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkMoveAttachments">
     <property name="toolTip">
      <string>Attachments will not be delete if checked</string>
     </property>
     <property name="text">
      <string>Move a file instead of copying when it is imported</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkAutoBack">
       <property name="text">
        <string>Auto Backup</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Keep</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbBackupDays">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>365</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Days</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>-25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAutoBack</sender>
   <signal>clicked(bool)</signal>
   <receiver>sbBackupDays</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
